types/peerjs/index.d.ts

```diff
@@ -2,9 +2,7 @@
 // Project: http://peerjs.com/
 // Definitions by: Toshiya Nakakura <https://github.com/nakakura>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
-
-/// <reference types="webrtc" />
+// TypeScript Version: 3.2
```

Comment:
I thought you had to use `typesVersion` when using any version `>= 3.1`

Reply:
No, that's only if you want to provide simultaneous 3.1+ and pre-3.1 support.

Reply:
Later: Well, looks like I'm wrong! The build fails when you require a version newer than 3.1 now. I guess I'll have to learn how to use typesVersions again.